.travis.yml

```diff
@@ -44,3 +44,20 @@ script:
 
 after_success:
     - php vendor/bin/coveralls -v
+
+before_deploy:
+    - 'sed -i "s/^{/\{\n    \"config\": {\"platform\": {\"php\": \"5.3.6\"}},/" composer.json'
+    - curl -LSs http://box-project.github.io/box2/installer.php | php
+    - composer update --no-dev --no-interaction --optimize-autoloader --prefer-stable
+    - php -d phar.readonly=false box.phar build
+
+deploy:
+    provider: releases
```

Comment:
Not sure to fully understand. Where Travis push the .phar? On the release downloads section like this?

![image](https://cloud.githubusercontent.com/assets/1698357/16389625/0a67ccda-3c9f-11e6-81f5-ebfa84626a34.png)

Or directly committed on the repository?